File: spring-boot-sample-common/src/main/java/com/github/jccode/springbootsample/core/model/BaseEntity.java
package com.github.jccode.springbootsample.core.model;

import java.io.Serializable;

public class BaseEntity implements Serializable {

    private Integer id;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
}

File: spring-boot-sample-common/src/main/scala/com/github/jccode/springbootsample/core/model/BaseEntity.scala
package com.github.jccode.springbootsample.core.model

import java.util.Date

import scala.beans.BeanProperty

case class BaseEntity (@BeanProperty var id: Int,
                       @BeanProperty var createTime: Date,
                       @BeanProperty var updateTime: Date
                      ) extends Serializable {

}
